ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+bash ci/codecov.sh
```

Comment:
mmm, this after success script is not running in a container at all :thinking: (beaver should take care of this so for now I will not focus on improving this), but I would put here:

```sh
curl -s https://codecov.io/bash | docker run -ti --rm -v PATH_TO_REPORTS ocean bash
```